testtracer_test.go

```diff
@@ -0,0 +1,48 @@
+package opentracing
+
+// testTracer is a most-noop Tracer implementation that makes it possible for
+// unittests to verify whether certain methods were / were not called.
+type testTracer struct{}
```

Comment:
a noopTracer fork, basically... obviously could do a lot more here, all I really needed at the moment was `testSpan.HasParent`